sycl/test/abi/layout_array.cpp

```diff
@@ -0,0 +1,84 @@
+// RUN: %clangxx -fsycl -c -fno-color-diagnostics -Xclang -fdump-record-layouts %s | FileCheck %s
+// REQUIRES: linux
+
+// clang-format off
+
+// TODO fix individual headers and include them instead of sycl.hpp
+#include <CL/sycl.hpp>
+
+
+void range(sycl::range<2>) {}
```

Comment:
Should we have these tests for 1-dim and 3-dim also?

Reply:
I'm not sure there's much sense in looking at different sizes, since specializing a template is unlikely to add, remove or reorder fields in structures. Do you have other opinion? Also tagging @romanovvlad

Reply:
Sounds reasonable. OK, then.

Reply:
If "conversation is resolved", please, mark it so, using GitHub UI.

Reply:
Sorry, I missed it. Resolved.